buildSrc/subprojects/basics/src/main/kotlin/gradlebuild/basics/tasks/ClasspathManifest.kt

```diff
@@ -52,18 +58,18 @@ abstract class ClasspathManifest : DefaultTask() {
 
     private
     fun createProperties() = Properties().also { properties ->
-        properties["runtime"] = externalDependencies.map { it.name }.joinForProperties()
-        properties["projects"] = runtimeClasspath.filter { it.isGradleModule() }.map { it.toGradleModuleName() }.joinForProperties()
+        properties["runtime"] = getRuntime().get().joinToString(",")
+        properties["projects"] = getProjects().get().joinToString(",")
         optionalProjects.get().takeIf { it.isNotEmpty() }?.let { optional ->
             properties["optional"] = optional.joinForProperties()
         }
     }
 
     private
-    fun File.isGradleModule() = name.startsWith("gradle-") || name.contains("-patched-for-gradle-")
+    fun FileSystemLocation.isGradleModule() = asFile.name.startsWith("gradle-") || name.contains("-patched-for-gradle-")
```

Comment:
Should be `|| asFile.name.contains("-patched-for-gradle-")`

Reply:
Thanks, fixed.